build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.2'
+        classpath 'com.android.tools.build:gradle:7.0.3'
```

Comment:
Is updating dependencies intended?

Reply:
No, the update of the dependencies was not intended, we just want to keet the same version of gradle as upstream
[https://github.com/TeamNewPipe/NewPipe/blob/4fc9443b4fb2ec481f9d9bcd9b0df045f928307d/build.gradle](url)

Reply:
If you want to do this, update your git branch from our upstream directly and don't replicate changes into yours. To do so, you will have to:

- add our repo as a git remote (in my commands, it is called `NEWPIPE_REMOTE_NAME`);
- fetch this remote (`git fetch NEWPIPE_REMOTE_NAME`);
- select your branch, drop the first commit (to remove this change) (you can do this by using `git rebase -i HEAD~NUMBER_OF_YOUR_COMMITS` and following instructions to remove the commit (you will need to put a `d` and a space before the SHA1 of the commit you want to remove then to save the commit message and to close your editor, if I am not wrong));
- rebase your branch into our `dev` branch (when your branch is selected, run `git rebase NEWPIPE_REMOTE_NAME/dev`);
- fix eventual conflicts and continue rebase;
- check if everything works as intended after the rebase;
- force push your changes your remote branch (the command line should be `git push -f origin magic_numbers_animation` in your case).

Reply:
@Novmbrain Could you remove these unrelated changes from upstream? Either like described or some other method.